### PR TITLE
Cleaning up variable name and logic

### PR DESCRIPTION
**Pull request by jazeee**
Since the increment counter has larger scope, I changed it from generic `i` to more identifiable name.
Also, simplified the logic.

### CODE REVIEW

File: lib/phantom_script.js [hunk 1]
```diff
@@ -66,17 +66,17 @@ var renderPage = function(url) {
     }
   });
 
-  var i = 0;
+  var renderIterations = 0
```
Comment (dr-dimitru) [inline lib/phantom_script.js]: Semicolon

File: lib/phantom_script.js [hunk 2]
```diff
@@ -66,17 +68,18 @@ var renderPage = function(url) {
     }
   });
 
-  var i = 0;
+  var renderIterations = 0;
```
Comment (jazeee) [inline lib/phantom_script.js]: Fixed.

File: lib/phantom_script.js [hunk 3]
```diff
@@ -66,17 +68,18 @@ var renderPage = function(url) {
...
   intervalId = setInterval(function() {
     var renderStatus = isReady();
-    if(status === null && i < 10 || status == 'null' && i < 10){
-      i++;
+    if(renderIterations < 40 && !renderStatus){
```
Comment (jazeee) [inline lib/phantom_script.js]: Also, addressed #13.  This seems to work better.  I can't 100% figure out what is the issue, but generally, I needed 3-4 seconds delay to ensure that this works reliably.

Comment (dr-dimitru) [inline lib/phantom_script.js]: `renderStatus` on some systems returns `'null'` as string while redirecting

File: lib/phantom_script.js [hunk 4]
```diff
@@ -66,17 +68,18 @@ var renderPage = function(url) {
     }
   });
 
-  var i = 0;
+  var renderIterations = 0;
   intervalId = setInterval(function() {
     var renderStatus = isReady();
-    if(status === null && i < 10 || status == 'null' && i < 10){
-      i++;
+    if(renderIterations < 40 && (!renderStatus || renderStatus === 'null')){
```
Comment (jazeee) [inline lib/phantom_script.js]: I added null check here, as requested, however, the documentation indicates that page.evaluate will return what we return in the function we give to evaluate, which is either true or false.
http://phantomjs.org/api/webpage/method/evaluate.html

Comment (dr-dimitru) [inline lib/phantom_script.js]: Sorry, not the `renderStatus`, but `status`, as it was on line 72

Comment (jazeee) [inline lib/phantom_script.js]: Ahh, OK.
Updated.

**Pull request merged**